src/Init/Util.lean

```diff
@@ -38,6 +38,41 @@ def dbgSleep {α : Type u} (ms : UInt32) (f : Unit → α) : α := f ()
 @[neverExtract, inline] def panicWithPosWithDecl {α : Type u} [Inhabited α] (modName : String) (declName : String) (line col : Nat) (msg : String) : α :=
   panic (mkPanicMessageWithDecl modName declName line col msg)
 
+/-!
+
+# The unsoundness of `ptrAddr`
+
+Reference: [Sealing Pointer-Based Optimizations Behind Pure Functions](https://arxiv.org/pdf/2003.01685.pdf)
```

Comment:
I'm not sure we should cite this preprint in the light of #1500.  And we should certainly avoid any claims of safety here.

Reply:
ah sorry I missed the memo on this